src/cli/definitions/parse-server.js

```diff
@@ -199,7 +199,7 @@ export default {
     help: "Run with cluster, optionally set the number of processes default to os.cpus().length",
     action: numberOrBoolParser("cluster")
   },
-   "liveQuery.classNames": {
```

Comment:
it was a duplicate of the next one, that would let you pass `--liveQuery.classNames` options easily